`make test` works very nicely

```bash
~/r/flam/helm-charts issue/2 ❯ make test                    timberhill@zenbook
Specify a branch for the dispatcher service:
BRANCH: main
Specify a branch for the submission service:
BRANCH: main
[+] Building 54.9s (27/27) FINISHED
......... logs and stuff here
```

`make up` doesn't know how to handle the lack of env variables if you run it first:

```bash
~/r/flam/helm-charts issue/2 ❯ make up
WARN[0000] The "SUBMISSION_BRANCH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DISPATCHER_BRANCH" variable is not set. Defaulting to a blank string. 
[+] Running 0/2
 ⠿ dispatcher Warning                                                     1.5s
 ⠿ submission Warning                                                     1.5s
[+] Building 0.9s (2/2) FINISHED                                               
 => [dispatcher-service internal] load remote build context               0.8s
 => CANCELED [submission-service internal] load git source https://githu  0.8s
failed to solve: dockerfile parse error on line 8: unknown instruction: <!DOCTYPE                                                                             
make: *** [Makefile:6: up] Error 17
```

But after `make test` created the files, it works fine, even though it lies about blank strings:

```bash
❯ make up
WARN[0000] The "DISPATCHER_BRANCH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SUBMISSION_BRANCH" variable is not set. Defaulting to a blank string. 
[+] Running 2/2
 ⠿ Container compose-submission-1  St...                                  0.6s
 ⠿ Container compose-dispatcher-1  St...                                  1.0s
```


Is it possible to make it default to `main` for instance?

--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,9 @@
 
 .PHONY: up
 up: ## Start services defined in compose/docker-compose.yaml
+##& Ensure you have the following set in your .env file:
+###& DISPATCHER_BRANCH=main
+###& SUBMISSION_BRANCH=main
 	@docker compose -f compose/docker-compose.yaml up -d
 
 .PHONY: down
@@ -23,11 +26,14 @@ test: ## Test a service from a given branch of its repo
 
 .PHONY: help
 help: ## Display this help
-###& Example: Additional information about help target usage
-	@echo "\nUsage:\n  make \033[36m<target>\033[0m"
+##& Examples:
+###& $ make help foo
+###& $ make help bar
+	@echo "\nUsage:\n  make \033[1m\033[36m<target>\033[0m"
 	@awk 'BEGIN {FS = ":.*##"}; \
-		/^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } \
+		/^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[1m\033[36m%-15s\033[0m %s\n", $$1, $$2 } \
 		/^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } \
 		/^###@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } \
-		/^###&/ { printf "\t\t  \033[33m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+		/^##&/ { printf "\t\t   \033[1m\033[33m%s\033[0m\n", substr($$0, 5) } \
+		/^###&/ { printf "\t\t    \033[0m\033[36m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 ##@ To add more information to the help output, see the example provided in help target

--- a/README.md
+++ b/README.md
@@ -23,15 +23,20 @@ _none yet_
 
 ## Docker compose
 
-Set Reddit credentials in a `.env` file (required by the [dispatcher](https://github.com/flam-flam/dispatcher-service)):
+### Pre-requisites
+docker version `>=23.0.0`
 
-```env
-REDDIT_CLIENT_ID=<value>
-REDDIT_CLIENT_SECRET=<value>
-```
+#### Steps:
+1. Create new or obtain existing Reddit credentials (required by the [dispatcher](https://github.com/flam-flam/dispatcher-service))
+2. Create a `.env` file:
+    ```env
+    REDDIT_CLIENT_ID=<value>
+    REDDIT_CLIENT_SECRET=<value>
+    DISPATCHER_BRANCH=main
+    SUBMISSION_BRANCH=main
+    ```
 
-Run the services in docker:
-
-```bash
-make up
-```
+3. Run the services in docker:
+    ```bash
+    make up
+    ```

